src/components/Table/Table.stories.tsx

```diff
@@ -0,0 +1,189 @@
+import { storiesOf } from '@storybook/react'
+import * as React from 'react'
+
+import styled from 'styled-components'
+
+import { Table } from './Table'
+import { Body } from './Body'
+import { Head } from './Head'
+import { Row } from './Row'
+import { Cell } from './Cell'
+import { action } from '@storybook/addon-actions'
+import { Icon } from '../Icon'
+import { Checkbox as CheckboxComponent } from '../Checkbox'
+import { Button } from '../Button'
+import { Base } from '../Base'
+
+import readme from './README.md'
+
+const data = [
+  {
+    name: 'Tea',
+    calories: 0,
+    fat: 0,
+    carbs: 0,
+    protein: 0,
+  },
+  {
+    name: 'Smoothie',
+    calories: 135,
+    fat: 0,
+    carbs: 34.5,
+    protein: 1.0,
+  },
+  {
+    name: 'Milk',
+    calories: 138,
+    fat: 7.8,
+    carbs: 9.9,
+    protein: 6.8,
+  },
+]
+
+storiesOf('Table', module)
+  .addParameters({
+    readme: {
+      sidebar: readme,
+    },
+  })
+  .add('all', () => (
+    <Ul>
+      <li>
+        table
+        <Table>
+          <Head>
+            <Row>
+              <Cell>
+                <Checkbox name="tableCheckbox" checked={false}></Checkbox>
+              </Cell>
+              <Cell onClick={action('clicked')}>
+                <ClickableCell>
+                  Name
+                  <Arrow name="fa-arrow-down" />
+                </ClickableCell>
+              </Cell>
+              <Cell>Calories</Cell>
+              <Cell>Fat (g)</Cell>
+              <Cell>Carbs (g)</Cell>
+              <Cell>Protein (g)</Cell>
+              <Cell>Button</Cell>
+            </Row>
+          </Head>
+          <Body>
+            {data.map(({ name, calories, fat, carbs, protein }) => {
+              return (
+                <Row key={name}>
+                  <Cell>
+                    <Checkbox name="tableCheckbox" checked={false}></Checkbox>
+                  </Cell>
+                  <Cell>{name}</Cell>
+                  <Cell>{calories}</Cell>
+                  <Cell>{fat}</Cell>
+                  <Cell>{carbs}</Cell>
+                  <Cell>{protein}</Cell>
+                  <Cell>
+                    <Button type="sub-a" size="s">
```

Comment:
Please stop using this button ( `type="sub-a"`).
This button design will be EOL soon.

We can use `type="sub-b"` instead of SecondaryButton.